CONTRIBUTING.md

```diff
@@ -39,11 +39,29 @@ issues on a regular basis; however note that sometimes this can take awhile
 issue and the work schedules of people involved--this does not mean we are
 ignoring the issue.
 
-For an issue that requires WG discussion or approval, WG members can tag/label
-it 'Agenda+' to bring it to the Working Group's attention. Unfortunately,
+When issues need WG discussion or approval, WG members should label the issue
+'Agenda+' to bring it to the Working Group's attention. Unfortunately,
 GitHub doesn't allow labelling permissions without repository write permissions,
 so if you believe an issue is urgent or discussion has stalled for awhile and
-the WG's attention is needed to move forward, ask one of the editors to flag it.
+the WG's attention is needed to move forward, ask one of the Editors to flag it.
+
+In general, you should not directly commit changes to a draft unless you are an
+Editor of that draft, or have their explicit permission.  If you are not an 
+Editor of a draft, but wish to contribute changes, the best practice is to either 
+work directly with an Editor to review proposed text, or file your proposal as a 
+pull request (PR). Substantive changes need WG consensus, not merely Editor 
+agreement; typographic error and markup fixes are generally okay to commit, but 
+any substantive changes, or even substantial changes/additions to non-normative
+text, should have clear WG consensus.
```

Comment:
Non-normative text doesn't generally require WG consensus, only if it's reflecting something sensitive. Substantial changes to non-normative text should have the editors approval though.

Reply:
Addressed.